compiler/src/Language/Bond/Codegen/Cpp/Util.hs

```diff
@@ -103,6 +103,7 @@ defaultValue _ _ _ = error "defaultValue: impossible happened."
 enumValue :: ToText a => MappingContext -> Type -> a -> Text
 enumValue cpp (BT_UserDefined e@Enum {..} _) x =
     [lt|#{getQualifiedName cpp $ getDeclNamespace cpp e}::_bond_enumerators::#{declName}::#{x}|]
+enumValue cpp (BT_UserDefined a@Alias {..} args) e = enumValue cpp (resolveAlias a args) e
```

Comment:
We should also update one of the golden tests (like [aliases.bond](https://github.com/Microsoft/bond/blob/master/compiler/tests/schema/aliases.bond)) to show how code gets generated for this construct and to make sure it changes only when we want it to.

To update the golden tests:
```
cd compiler
cabal test # expect this to fail
dist/build/gbc-tests/gbc-tests --accept
cabal test # this should now pass
git add -u :/
# Other git commands to update the PR
```